tools/proteomiqon_mzmltomzlite/proteomiqon_mzmltomzlite.xml

```diff
@@ -0,0 +1,292 @@
+            <when value="Centroid">
+                <conditional name="CentroidModeCond">
+                    <param name="CentroidMode" type="select" label="Centroidization mode">
+                        <option value="Manufacturer">Manufacturer</option>
+                        <option value="Wavelet">Wavelet</option>
+                    </param>
+                    <when value="Wavelet">
+                        <param type="integer" name="NumberOfScales" label="Number of scales" value="0"/>
+                        <conditional name="YThresholdCond">
+                            <param name="YThreshold" type="select" label="Y threshold">
+                                <option value="Fixed" selected="true">Fixed</option>
+                                <option value="MinSpectrumIntensity">Min spectrum intensity</option>
+                            </param>
+                            <when value="Fixed">
+                                <param type="float" name="Ms2YThreshold" label="Value" value="0"/>
+                            </when>
+                        </conditional>
+                        <param type="float" name="Centroid_MzTolerance" label="Centroid MZ tolerance" value="0"/>
+                        <param type="float" name="SNRS_Percentile" label="SNRS percentile" value="0"/>
+                        <param type="float" name="MinSNR" label="Min SNR" value="0"/>
+                        <param type="boolean" name="RefineMZ" label="Refine MZ" value="false"/>
+                        <param type="boolean" name="SumIntensities" label="Sum intensities" value="false"/>
+                        <conditional name="PaddingParamsCond">
+                            <param type="boolean" name="usePaddingparams" label="Padding" checked="false"/>           
+                            <when value="true">
+                                <conditional name="MaximumPaddingPointsCond">
+                                    <param type="boolean" name="specifyMaximumPaddingPoints" label="Speficy maximum padding points" checked="true"/>
+                                    <when value="true">
+                                        <param type="integer" name="MaximumPaddingPoints" label="Maximum padding points" value="0"/>
+                                    </when>        
+                                </conditional>
+                                <param type="float" name="Padding_MzTolerance" label="Padding MZ tolerance" value="0"/>
+                                <param type="integer" name="WindowSize" label="Window size" value="0"/>
+                                <param type="float" name="SpacingPerc" label="Spacing perc" value="0"/>
+                            </when>
+                        </conditional>
+                    </when>
+                </conditional>
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data format="sqlite" name="out_mzlite" />
+    </outputs>
```

Comment:
please add tests, we need to test all tools before they can enter the main Galaxy servers

That one might help with formatting, bootstrapping tests and much more: https://github.com/galaxyproject/galaxy-language-server